CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,11 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect everyone who contributes by posting issues, updating documentation, submitting pull requests, providing feedback in comments, and any other activities.
+
+Communication through any of this project's channels (GitHub, Slack, IRC, mailing lists, Twitter, etc.) must be constructive and never resort to personal attacks, trolling, public or private harassment, insults, or other disrectpectful conduct.
+
+We promise to extend courtesy and respect to everyone involved in this project regardless of gender, gender identity, sexual orientation, disability, age, race, ethnicity, religion, or level of experience. We expect anyone contributing to this project to do the same.
+
+If any member of the community violates this code of conduct, the maintainers of this project will take action, removing issues, comments, and PRs or blocking accounts as deemed appropriate.
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please contact us using any of this project's channels.
```

Comment:
Wonder if we would need a sort of team@chatboot.ai alias to be contacted

Reply:
It's usually bet to a contact email yes, we can always add it later (like when we have the domain 😉 )